config.py

```diff
@@ -29,7 +29,7 @@
 # appversion **MUST** follow Semantic Versioning rules:
 # <https://semver.org/#semantic-versioning-specification-semver>
 # Major.Minor.Patch(-prerelease)(+buildmetadata)
-appversion = '5.0.0-beta1'  # -rc1+a872b5f'
+appversion = '4.2.0-beta1'  # -rc1+a872b5f'
```

Comment:
I think this might be a holdover we don't want

Reply:
The one time I picked the wrong option resolving a rebase, thanks.